skywalking/plugins/__init__.py

```diff
@@ -38,7 +41,65 @@ def install():
             continue
         logger.debug('installing plugin %s', modname)
         plugin = importer.find_module(modname).load_module(modname)
+
+        logger.debug('checking version for plugin %s', modname)
+        supported = pkg_version_check(plugin, modname)
+        if not supported:
+            continue
```

Comment:
[1] move `logger.debug('checking version for plugin %s', modname)` into the `if not supported` block, and reword the message to tell the users that the current version is not supported by the plugin